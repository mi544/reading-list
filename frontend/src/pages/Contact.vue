<template>
  <div class="flex-grow flex items-center justify-center text-2xl">
    So here we are... Contact page!
  </div>
</template>

<script>
export default {
  name: 'Contact',
}
</script>
