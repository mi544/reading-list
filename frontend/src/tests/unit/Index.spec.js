/* eslint-disable no-undef */
import { shallowMount } from '@vue/test-utils'
import Contact from '../../pages/Contact.vue'

describe('Contact', () => {
  it('mounts successfully', () => {
    const wrapper = shallowMount(Contact)
    expect(wrapper.exists()).toBeTruthy()
  })
  it('matches snapshot', () => {
    const wrapper = shallowMount(Contact)
    expect(wrapper.element).toMatchSnapshot()
  })
})
