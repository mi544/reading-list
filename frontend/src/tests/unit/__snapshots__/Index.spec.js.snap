// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact matches snapshot 1`] = `
<main
  class="flex-grow flex items-center justify-center text-2xl"
>
   So here we are... Contact page! 
</main>
`;
